RELEASES.md

```diff
@@ -90,43 +85,54 @@ whichever is longer. Additionally, releases may have an extended security suppor
-| [2.0](https://github.com/containerd/containerd/releases/tag/v2.0.0)  | Active        | November 5, 2024   | max(November 5, 2025 or release of 2.1 + 6 months)      |
-| [2.1](https://github.com/containerd/containerd/milestone/48)         | Next          | TBD                | TBD                                                     |
-
-> **_NOTE_** containerd v1.7 will end of life at the same time as v1.6 LTS. Due to
-> [Minimal Version Selection](https://go.dev/ref/mod#minimal-version-selection) used
-> by Go modules, 1.7 must be supported until EOL of all 1.x releases. Once 1.7 is in
-> extended support, it will continue to accept security patches in addition to client
-> changes relevant for package importers using the 1.6 LTS daemon.
+Long term stable (_LTS_) releases are sponsored by at least two maintainers for at least two
+years after their initial _minor_ (x.y.0) release. The maintainers of the _LTS_ branch may commit to
+a longer period or extend the support period as needed. These branches will accept bug reports and
+backports until the end of life date. They may also accept a wider range of patches than non-_LTS_
+releases to support the longer term maintainability of the branch, including library dependency,
+toolchain (including Go) and other version updates which are needed to ensure each release is built
+with fully supported dependencies. Feature backports are up to the discretion of the maintainers who
+own the branch but should be rejected by default. There is no defined limit to the number of _LTS_
+branches and any branch may become an _LTS_ branch after its initial release. There is no guarantee
+that a new _LTS_ branch will be designated before existing _LTS_ branches reach their end of life.
+
+### Release Owners
+
+Every release shall be assigned ownership once it reaches beta. The initial release owners
+will act as the release captains and will be responsible for creating the releases and ensuring
+the release is on time. Once the release is in rc, the release owners should be part of any
+discussion around merging impactful or risky changes. Every release should have two owners who
+are both active maintainers and one of which has been involved in two prior releases.
+
+Once the final release is out and the release branch moves to active, ownership will be
+transferred back over to all committers. Active releases are maintained by all committers
+until the release reaches end of life or the branch transitions to _LTS_ with at least
+two maintainers keeping ownership.
+
+The owners of the _LTS_ branch may step down or be replaced by another maintainer at any time if
+they can no longer support the branch. If no maintainers volunteer to the own a branch after
+maintainers step down, the branch will end of life after 6 months of extended support with
+ownership transferred back to all committers.
+
+
+### All Releases
```

Comment:
maybe we should add column for LTS's sponsor?

Reply:
Thats what the owners field is intended for